FEATURES.md

```diff
@@ -0,0 +1,187 @@
+# Features 📈
+
+### Table of contents
+
+* [Main](#Main)
+* [Discover Stocks](#Discover_Stocks)
+* [Market Sentiment](#Market_Sentiment)
+* [Research Web pages](#Research_Web_pages)
+* [Fundamental Analysis](#Fundamental_Analysis)
+* [Technical Analysis](#Technical_Analysis)
+* [Due Diligence](#Due_Diligence)
+* [Prediction Techniques](#Prediction_Techniques)
+
+### Main <a name="Main"></a>
+
+The main menu allows the following commands:
+```
+load -t S_TICKER [-s S_START_DATE] [-i {1,5,15,30,60}]
+```
+   * Load stock ticker to perform analysis on
+     * -s : The starting date (format YYYY-MM-DD) of the stock
+     * -i : Intraday stock minutes
+```
+clear
+```
+   * Clear previously loaded stock ticker.
+```
+view -t S_TICKER [-s S_START_DATE] [-i {1,5,15,30,60}] [--type N_TYPE]
+```
+   * Visualise historical data of a stock. An alpha_vantage key is necessary.
+     * -s : The starting date (format YYYY-MM-DD) of the stock
+     * -i : Intraday stock minutes
+     * --type : 1234 corresponds to types: 1. open; 2. high; 3.low; 4. close; while 14 corresponds to types: 1.open; 4. close
+
+![GNUS](https://user-images.githubusercontent.com/25267873/108925137-f2920e80-7633-11eb-8274-6e3bb6a19592.png)
+
+Note: Until a ticker is loaded, the menu will only show *disc* and *sen* menu, as the others require a ticker being provided.
+
+### [Discover Stocks](discovery/README.md) <a name="Discover_Stocks"></a>
+Command|Description|Source
+---|---|---
+`map`           |S&P500 index stocks map |[Finviz](https://finviz.com)
```

Comment:
Are you going to leave all these tables?  

The main commands are in a different format.

Reply:
Yes, that's the point. Because the main commands should have a better explanation in this README.

The other commands are like a "quick functionality show", but if you follow through their README, they'll have a more detailed explanation just like the main commands have in this file.

Does that make sense?